bd-workflows/src/sankey_diagram.rs

```diff
@@ -197,24 +205,27 @@ impl Processor {
       .await
     {
       log::debug!("failed to send sankey upload request: {e}");
-      self.stats.upload_completion_failure_total.inc();
+      self.stats.upload_completion_failures.inc();
       return;
     }
 
     if let Err(error) = response.await {
       log::debug!("failed to wait for sankey upload response: {error}");
-      self.stats.upload_completion_failure_total.inc();
+      self.stats.upload_completion_failures.inc();
       return;
     }
 
-    self.stats.upload_completion_success_total.inc();
+    // Augustyniak: Is it risky to insert the path only in the case of a successful upload? maybe we
```

Comment:
What's the consequence of this? Are you worried that we would keep sending intents or is that covered elsewhere?